lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -107,9 +107,9 @@ def extract_method(node)
 
         # Only add the method to the chain if the method being
         # called if it's a member of the time class.
-        def add_to_chain(node)
+        def receiver_contains_time?(node)
           receiver, _method_name, *_args = *node
-          !(receiver.inspect.index 'Time').nil?
+          receiver.source.include? 'Time' 
```

Comment:
Won't this be fooled by something like `Timetest` or whatever? Won't `Time` normally be at the root of the method chain? Why don't you check if the root receiver is time by drilling down the `send` node?